test/e2e/data/tiles/osmtile/2/1_1.mapml

```diff
@@ -0,0 +1,21 @@
+<mapml- xmlns="http://www.w3.org/1999/xhtml">
+    <map-head>
+        <map-title>ne_10m_admin_0_countries</map-title>
+        <map-meta charset="utf-8"/>
+        <map-meta content="text/mapml" http-equiv="Content-Type"/>
+      <map-meta name="projection" content="OSMTILE"></map-meta>
+      <map-meta name="cs" content="pcrs"></map-meta>
+    </map-head>
+    <map-body>
+        <map-feature id="fclass.73" class="fclass">
+            <map-geometry>
+                <map-polygon>
+                    <map-coordinates>
```

Comment:
You could put an xml comment here that says e.g. `<!-- leading and trailing whitespace should not affect parsing of map-coordinates -->